dev/service-registry/play-integration-javadsl/src/main/scala/com/lightbend/lagom/play/LagomPlayModule.scala

```diff
@@ -4,58 +4,33 @@
 package com.lightbend.lagom.play
 
 import java.net.URI
-import java.util.Optional
-
-import scala.collection.JavaConverters._
-import scala.compat.java8.FutureConverters._
-import com.google.inject.Provider
-import com.lightbend.lagom.javadsl.api.ServiceAcl
-import com.lightbend.lagom.javadsl.api.ServiceInfo
-import com.lightbend.lagom.javadsl.api.transport.Method
 import javax.inject.Inject
-import javax.inject.Singleton
 
 import com.lightbend.lagom.internal.javadsl.registry.{ ServiceRegistry, ServiceRegistryService }
-import play.api.Configuration
-import play.api.Environment
-import play.api.inject.ApplicationLifecycle
-import play.api.inject.Binding
-import play.api.inject.Module
+import com.lightbend.lagom.javadsl.api.ServiceInfo
+import play.api.inject.{ ApplicationLifecycle, Binding, Module }
+import play.api.{ Configuration, Environment }
+
+import scala.collection.JavaConverters._
+import scala.compat.java8.FutureConverters._
 
 class LagomPlayModule extends Module {
   override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] = {
     Seq(
-      bind[PlayRegisterWithServiceRegistry].toSelf.eagerly(),
-      bind[ServiceInfo].toProvider[PlayServiceInfoProvider]
+      bind[PlayRegisterWithServiceRegistry].toSelf.eagerly()
```

Comment:
This is a breaking change - I think we should provide backwards compatibility here. If there is a lagam.play.service-name defined in configuration, then bind PlayServiceInfoProvider, and output a warning saying that binding the service name from configuration is deprecated, otherwise don't.